lib/AST/RawComment.cpp

```diff
@@ -247,3 +247,13 @@ StringRef Decl::getBriefComment() const {
   Context.setBriefComment(this, Result);
   return Result;
 }
+
+CharSourceRange RawComment::getCharSourceRange(const SourceManager &SM) {
+  if (this->isEmpty()) {
+    return CharSourceRange();
+  }
+
+  auto Start = this->Comments.front().Range.getStart();
```

Comment:
Could we have an early return if `Start` is invalid? cause the doc comment may come from a deserialized module file.